src/TextUI/TestRunner.php

```diff
@@ -1111,50 +1151,53 @@ protected function handleConfiguration(array &$arguments): void
+        $arguments['processUncoveredFilesFromWhitelist']                       = $arguments['processUncoveredFilesFromWhitelist'] ?? false;
+        $arguments['randomOrderSeed']                                          = $arguments['randomOrderSeed'] ?? \time();
+        $arguments['registerMockObjectsFromTestArgumentsRecursively']          = $arguments['registerMockObjectsFromTestArgumentsRecursively'] ?? false;
+        $arguments['repeat']                                                   = $arguments['repeat'] ?? false;
+        $arguments['reportHighLowerBound']                                     = $arguments['reportHighLowerBound'] ?? 90;
+        $arguments['reportLowUpperBound']                                      = $arguments['reportLowUpperBound'] ?? 50;
+        $arguments['reportUselessTests']                                       = $arguments['reportUselessTests'] ?? true;
+        $arguments['reverseList']                                              = $arguments['reverseList'] ?? false;
+        $arguments['executionOrder']                                           = $arguments['executionOrder'] ?? TestSuiteSorter::ORDER_DEFAULT;
+        $arguments['resolveDependencies']                                      = $arguments['resolveDependencies'] ?? false;
+        $arguments['stopOnError']                                              = $arguments['stopOnError'] ?? false;
+        $arguments['stopOnFailure']                                            = $arguments['stopOnFailure'] ?? false;
+        $arguments['stopOnIncomplete']                                         = $arguments['stopOnIncomplete'] ?? false;
+        $arguments['stopOnRisky']                                              = $arguments['stopOnRisky'] ?? false;
+        $arguments['stopOnSkipped']                                            = $arguments['stopOnSkipped'] ?? false;
+        $arguments['stopOnWarning']                                            = $arguments['stopOnWarning'] ?? false;
+        $arguments['stopOnDefect']                                             = $arguments['stopOnDefect'] ?? false;
+        $arguments['strictCoverage']                                           = $arguments['strictCoverage'] ?? false;
+        $arguments['testdoxExcludeGroups']                                     = $arguments['testdoxExcludeGroups'] ?? [];
+        $arguments['testdoxGroups']                                            = $arguments['testdoxGroups'] ?? [];
+        $arguments['timeoutForLargeTests']                                     = $arguments['timeoutForLargeTests'] ?? 60;
+        $arguments['timeoutForMediumTests']                                    = $arguments['timeoutForMediumTests'] ?? 10;
+        $arguments['timeoutForSmallTests']                                     = $arguments['timeoutForSmallTests'] ?? 1;
+        $arguments['verbose']                                                  = $arguments['verbose'] ?? false;
```

Comment:
I do not understand this whitespace change.

Reply:
Me neither, to be honest. I have just removed all whitespace between `]` and `=` and had it reformatted. Looks like the assignment alignment doesn't remove excess whitespace once it has been introduced. Will commit a WS fix